server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationMemberDao.java

```diff
@@ -55,4 +58,21 @@ public void deleteByOrganizationUuid(DbSession dbSession, String organizationMem
     return mapper(dbSession).selectOrganizationUuidsByUser(userId);
   }
 
+  /**
+   *
+   * @param loginOrganizationConsumer {@link BiConsumer}<String,String> (login, organization uuid)
```

Comment:
I want to specify the first and second element of the BiConsumer (login, organization uuid). Otherwise, it's not obvious